test/spec/oauthUtil.js

```diff
@@ -1,6 +1,54 @@
 define(function(require) {
   var OktaAuth = require('OktaAuth');
   var oauthUtil = require('../../lib/oauthUtil');
+  var util = require('../util/util');
+  var wellKnown = require('../xhr/well-known');
+
+  describe('getWellKnown', function() {
+    util.itMakesCorrectRequestResponse({
```

Comment:
It also seems like we probably need a test for not using the cached item if it exceeds the time when it expired (1 day)